examples/01a-Getting-started.ipynb

```diff
@@ -0,0 +1,632 @@
+{
```

Comment:
<p>Do we need to install unzip? How does it work in this example: </p><p>https://github.com/NVIDIA-Merlin/NVTabular/blob/main/examples/getting-started-movielens/01-Download-Convert.ipynb </p><p> </p><p>I think we should not install libraries in the example. We should make a note, that it requires unzip (maybe download_file function doesnt rely on unzip)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/dataloader/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>everything works now with just <code>download_file</code> without a need to install anything else 👍 </p>